### PR TITLE
az-account, az-appconfig, az-bicep, az-config, az-devops, az-feedback: add Spanish translation

### PR DESCRIPTION
**Pull request by crixodia**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kant — APPROVED** [review]
LGTM

**Review by blueskyson — APPROVED** [review]

File: pages.es/common/az-account.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+> Administra la información de una suscripción de Azure.
+> Parte de `az`, el cliente de línea de comandos para Microsoft Azure.
+> Más información: <https://learn.microsoft.com/cli/azure/account>.
+
+- Lista las suscripciones de la cuenta activa:
+
+`az account list`
+
+- Establece una `suscripción` como la suscripción activa:
```
Comment (kbdharun) [inline pages.es/common/az-account.md:11]:
```suggestion
- Establece una `subscription` como la suscripción activa:
```
Note: the subcommand here shouldn't be translated.

File: pages.es/common/az-account.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+`az account set --subscription {{id_de_suscripción}}`
+
+- Lista las regiones admitidas para la suscripción activa:
+
+`az account list-locations`
+
+- Imprime un token de acceso para usar con la `API de MS Graph`:
```
Comment (kbdharun) [inline pages.es/common/az-account.md:19]:
```suggestion
- Imprime un token de acceso para usar con la `MS Graph API`:
```

File: pages.es/common/az-bicep.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+`az bicep install`
+
+- Crear un archivo de Bicep:
+
+`az bicep build --file {{ruta/al/archivo.bicep}}`
+
+- Intenta descompilar un archivo de plantilla ARM a un archivo de Bicep:
+
+`az bicep decompile --file {{ruta/al/archivo.json}}`
```
Comment (kbdharun) [inline pages.es/common/az-bicep.md:17]:
```suggestion
`az bicep decompile --file {{ruta/al/archivo_plantilla.json}}`
```
Shouldn't it be something like `archivo_plantilla` for `template_file`?

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by kant — COMMENTED** [review]
LGTM

**Pull request merged**